terasoluna-gfw-parent/pom.xml

```diff
@@ -458,6 +458,17 @@
         <groupId>com.sun.mail</groupId>
         <artifactId>javax.mail</artifactId>
         <version>${javax-mail.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>javax.activation</groupId>
+            <artifactId>activation</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>com.sun.activation</groupId>
+        <artifactId>javax.activation</artifactId>
+        <version>1.2.0</version>
```

Comment:
- [ ] Please use `properties`!

Reply:
d09941ab9ba67327dd976c0c6c59fe64e05f3225

I changed to use `properties` .